cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,28 @@
+import { generateUser } from '../support/generate';
+
+Cypress.on('uncaught:exception', () => {
+  return false;
+});
+
 describe('Sign Up page', () => {
   it('should provide an ability to register new account', () => {
+    const { email, username, password } = generateUser();
+
+    cy.visit('/#/register?_k=spysa9');
+
+    cy.get('[placeholder=Username]')
+      .type(username);
+
+    cy.get('[placeholder=Email]')
+      .type(email);
+
+    cy.get('[placeholder=Password]')
+      .type(password);
+
+    cy.get('button[type=submit]').click();
+
+    cy.url().should('contain', '/#/?_k=c1q9kl');
```

Comment:
It's better to avoid hardcoding URLs in your tests. If the URL structure changes, your tests will break. Consider using a relative URL or a URL from the application's state.